### ISSUE TITLE
Is Examples/VideoPlayer completely broken now?

### ISSUE THREAD

**Issue by PublicParadise** [TheWidlarzGroup/react-native-video#117]
Run this:

``` bash
git clone https://github.com/brentvatne/react-native-video.git
cd react-native-video
npm install
cd Example/VideoPlayer
npm install
```

You'll get:

```
npm ERR! git rev-list -n1 feature/android-support: fatal: ambiguous argument 'feature/android-support': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 feature/android-support: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 feature/android-support: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 feature/android-support: 
npm ERR! git rev-list -n1 feature/android-support: fatal: ambiguous argument 'feature/android-support': unknown revision or path not in the working tree.
...
```

Those errors are probably coming from this line in `package.json`:

``` json
"react-native-video": "brentvatne/react-native-video#feature/android-support"
```

**Comment by brentvatne** [comment]
oh, you can change that to master in the package.json -- we deleted that branch. PR welcome, thanks!

**Comment by PublicParadise** [comment]
@brentvatne Oh, there is more...

On ios I ran this:

``` bash
git clone https://github.com/brentvatne/react-native-video.git
cd react-native-video
react-native-video.git
cd react-native-video
npm install
cd Example/VideoPlayer
npm install react-native
open VideoPlayer.xcodeproj
```

In XCode I see that the project is incoherent:

<img width="262" alt="screen shot 2015-12-20 at 4 25 01 pm" src="https://cloud.githubusercontent.com/assets/2541332/11920487/b4b7015a-a736-11e5-82a7-85338bdd39d3.png">

I am seeing two problems:
- `VideoPlayer.xcodeproj` includes `node_modules/react-native-video/RCTVideo.xcodeproj`, which should reference the source files from `.../react-native-video/Example/VideoPlayer/../../RCTVideo.h` etc.
- what's the deal with `libicucore.dylib`?

**Comment by PublicParadise** [comment]
@brentvatne Wait, there is more!

After fixing the incorrect references in `Examples/VideoPlayer/node_modules/react-native-video/RCTVideo.xcodeproj` and hitting run I am getting this puppy:

<img width="620" alt="screen shot 2015-12-20 at 4 34 14 pm" src="https://cloud.githubusercontent.com/assets/2541332/11920514/905dfa6a-a737-11e5-9ad2-90c2c148ad7e.png">

That looks like `index.ios.js` hasn't been updated in a long time. I suspect this error is about https://github.com/magus/react-native-facebook-login/issues/31

**Comment by PublicParadise** [comment]
@brentvatne What has happened to this project? It was in a fairly good shape the last time I played with it (~July 2015). My PR from June (https://github.com/brentvatne/react-native-video/pull/68) is still in the freezer? 

Has this project become abandonware?

**Comment by PublicParadise** [comment]
In the XCode terminal I am seeing:

```
2015-12-20 16:43:26.634 [error][tid:main] Unable to resolve module react-native-video from /Users/bparadie/git/research/react/react-native-video/Examples/VideoPlayer/index.ios.js: Invalid directory /Users/node_modules/react-native-video
```

Adding these lines to `index.ios.js` didn't help:

```
  NativeModules,
  requireNativeComponent,
```

**Comment by PublicParadise** [comment]
My environment:

```
OSX 10.10.5
XCode 7.2
npm --version # 2.14.15
node --version # v5.3.0
watchman --version #3.8.0
```

**Comment by PublicParadise** [comment]
@brentvatne Ok, I got the example not working again. 
Here are the steps:
- fix package.json:

```
  "dependencies": {
    "react-native": "^0.16.0",
    "react-native-video": "0.6.1"
  }
```
- fix `node_modules/react-native-video/RCTVideo.xcodeproj`, let sources point to `../..`

**Comment by PublicParadise** [comment]
I updated https://github.com/brentvatne/react-native-video/pull/68 and added a fix for `package.json`.
Any takers?

**Comment by PublicParadise** [comment]
@brentvatne There were a bunch of things wrong with the `VideoPlayer` example. Just adding a version number to the `react-native-video` dependency in `package.json` wasn't enough, because that doesn't allow you to test `VideoPlayer` against your local changes.
Either way, I cleaned everything up in https://github.com/brentvatne/react-native-video/pull/118.

Now, may I hope that this PR will be looked at? I am thinking of https://github.com/facebook/react-native/pull/1546 where I spent a lot of time fine tuning the PR and nobody looked at it until I had to assume that there was no interest in it. So I closed it...
